### ISSUE TITLE
Display the graph after it is stabilized

### ISSUE THREAD

**Issue by arunkumarpro1** [crubier/react-graph-vis#25]
Hi,

The graph is taking some time to stabilize after being rendered. But I want the network to be stabilized before being displayed. I tried using the stabilization options under the physics module, but I could not achieve it.

This is my options

`let options = {
	physics:{
	 enabled: true,
		 stabilization: {
			 iterations:5000
			 }
	},
    layout: {
        hierarchical: false
    },
    nodes : {
    shape : 'box'
    },
    autoResize: false,
    edges: {
      smooth: false,
      color: '#000000',
      width: 0.5,
      arrows: {
        to: {
          enabled: true,
          scaleFactor: 0.5
        }
      }
    },
    interaction:{
	    hover : true,
	    keyboard :true,
	    navigationButtons :true,
	    tooltipDelay: 150
	  }
};`

can anyone help me with this?
Thanks in advance.

**Comment by crubier** [comment]
Hi, you should probably use the `stabilized` event of the underlying vis.js instance. See the doc there: http://visjs.org/docs/network/#Events .

```jsx

var events = {
    stabilized: function(event) {
        var { iterations } = event;
        // Here you should make the graph visible
    }
}

React.render(<Graph graph={graph} options={options} events={events} />, document.body);
```

**Comment by arunkumarpro1** [comment]
Thanks for the reply.

But isn't that the default behavior in vis networks. 

When I tried creating a separate vis network using the following option object, I could achieve that behavior.

``var options = {
                    physics: {
                   stabilization: {
                      enabled: true,
                      iterations: 5000    
                  }
                  }``

But it does not work in reactGraph.

**Comment by vedranserences** [comment]
Same problem here. Network stabilization is not working. Not matter what properties I put in options network id displayed before stabilization.

**Comment by brysgo** [comment]
Would love to see this become the default behavior

**Comment by crubier** [comment]
@vedranserences Can you do a PR with your proposed modifications ? I would like to review this

**Comment by crubier** [comment]
Should be solved in the latest version 1.0.7
